src/OpenColorIO/ParseUtils.cpp

```diff
@@ -257,14 +257,15 @@ const char * GpuLanguageToString(GpuLanguage language)
 {
     switch(language)
     {
-        case GPU_LANGUAGE_CG: return "cg";
-        case GPU_LANGUAGE_GLSL_1_2:  return "glsl_1.2";
-        case GPU_LANGUAGE_GLSL_1_3:  return "glsl_1.3";
-        case GPU_LANGUAGE_GLSL_4_0:  return "glsl_4.0";
+        case GPU_LANGUAGE_CG:           return "cg";
+        case GPU_LANGUAGE_GLSL_1_2:     return "glsl_1.2";
+        case GPU_LANGUAGE_GLSL_1_3:     return "glsl_1.3";
+        case GPU_LANGUAGE_GLSL_4_0:     return "glsl_4.0";
         case GPU_LANGUAGE_GLSL_ES_1_0:  return "glsl_es_1.0";
         case GPU_LANGUAGE_GLSL_ES_3_0:  return "glsl_es_3.0";
-        case GPU_LANGUAGE_HLSL_DX11: return "hlsl_dx11";
-        case LANGUAGE_OSL_1: return "osl_1";
+        case GPU_LANGUAGE_HLSL_DX11:    return "hlsl_dx11";
+        case GPU_LANGUAGE_MSL_2_0:      return "msl_metal";
```

Comment:
Please also update this to `msl_2`

Reply:
oops... good catch... sorry I missed that. Thanks for pointing it out.

Also, we decided to go with shading language name instead of shading language and api name.